concerned-coyotes/earlyinternet/utils/helpers.py

```diff
@@ -0,0 +1,10 @@
+def create_url_with_parameters(base_url: str, params: dict) -> str:
```

Comment:
Technically the name is misleading as you are creating a url with a query string not parameters